src/transform.ts

```diff
@@ -134,6 +137,21 @@ module.exports = exportedTokens;`,
   };
 }
 
+function processPostCSS(src: string, filename: string): TransformedSource {
+  return {code: `
+const postcss = require("postcss");
+const cssSrc = ${JSON.stringify(src)};
+postcss([require("tailwindcss")(), require("autoprefixer")()])
```

Comment:
Actually, I want to use [postcss-load-config](https://github.com/postcss/postcss-load-config) but it can't because it seems that this code template will run on the browser environment so the module gives us itself as ESModule that Jest can't parse.